### ISSUE TITLE
Hovering/clicking the number in the "Wrap at" option in Preferences makes the arrow buttons disappear

### ISSUE THREAD

**Issue by ItielMaN** [vslavik/poedit#367]
My environment:
Windows 10 build 1607
Poedit 2.0 stable, **Hebrew locale**
Monitor 24", screen resolution 1920x1080, DPI set to 150%

Before hovering the "Wrap at" field:
![default](https://cloud.githubusercontent.com/assets/18016656/24512925/8c98012c-1578-11e7-8a21-263708c9bab9.png)

After clicking/hovering the "Wrap at" field:
![default](https://cloud.githubusercontent.com/assets/18016656/24512995/babc6282-1578-11e7-90f6-6d4d1b57679a.png)

It looks like there's also an extra white space in that field.

It seems the hebrew version isn't the only locale suffering from this, I tested also Arabic and it's the same there, so I suspect this is a RTL issue.
This worked okay in 1.8.

**Comment by vslavik** [comment]
More interesting question: why is it partially offscreen and clearly misrendered?

**Comment by ItielMaN** [comment]
Yeah, that's exactly what I meant by "It looks like there's also an extra white space in that field". I think this is the souce of the issue.

**Comment by ItielMaN** [comment]
This appears to be fixed.
